install.md

```diff
@@ -67,6 +67,10 @@ load fixtures:
 
     (perma)vagrant@perma:/vagrant/perma_web$ fab dev.init_db
 
+You will also need the node requirements to compile frontend assets:
```

Comment:
Is this decision to keep the size of the Vagrant box small? (That is, python requirements are small enough, node requirements are too big?)